src/app/mat-select-search/mat-select-search.component.ts

```diff
@@ -366,6 +375,22 @@ export class MatSelectSearchComponent implements OnInit, OnDestroy, AfterViewIni
     }
   }
 
+  /**
+   * Handles the key up event with MatSelect.
+   * Allows e.g. the announcing of the currently activeDescendant by screen readers.
+   * @param event
+   */
+  _handleKeyup(event: KeyboardEvent) {
+    // Filter out all events that are not up arrow(38) or down arrow(40)
+    if (event.keyCode === 38 || event.keyCode === 40) {
```

Comment:
```suggestion
    if (event.keyCode === UP_ARROW || event.keyCode === DOWN_ARROW) {
```
these symbols can be imported from `@angular/cdk`

https://github.com/angular/components/blob/27fae292721bcc48b68c16fe5c7ac5e928eaa57d/src/cdk/keycodes/keycodes.ts#L26-L28

Reply:
:) i was looking for these, thanks